src/pages/home/platform-carousel/_mobile-platform-carousel.tsx

```diff
@@ -198,6 +206,18 @@ const MobilePlatformCarousel = ({ carousel_data }: MobilePlatformCarouselProps)
                             </Flex>
                             {title === 'Deriv MT5' ? (
                                 <OsBadges>
+                                    <AppStoreBadge
+                                        external
+                                        to={deriv_mt5_app_url}
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                    >
+                                        <QueryImage
+                                            data={data['dmt5_mobile_web_browser']}
+                                            alt="dmt5 web browser"
+                                        />
```

Comment:
Regarding my previous comment...
While we using `graphql` queries and `QueryImage` it's more or less good, we can keep this approach